enterprise/app/models/copilot_message.rb

```diff
@@ -38,13 +38,24 @@ def push_event_data
   end
 
   def enqueue_response_job(conversation_id, user_id)
-    Captain::Copilot::ResponseJob.perform_later(
-      assistant: copilot_thread.assistant,
-      conversation_id: conversation_id,
-      user_id: user_id,
-      copilot_thread_id: copilot_thread.id,
-      message: message['content']
-    )
+    if ENV['USE_AGENTS'] == 'true'
```

Comment:
should we do feature flag / installation config ?

Reply:
Agree. This is more of an account level setting.